relay-common/src/metrics.rs

```diff
@@ -64,30 +65,96 @@ where
     })
 }
 
+/// Any type that implements this trait can be set as a timer
+/// A typical implementation would be something like this
```

Comment:
Move this documentation into module level, and then you can refer to the module level docs on the traits like this:

```rust
/// Any type that implements this trait can be set as a counter.
///
/// See [module-level docs](index.html) for more information.
trait TimerMetric {}
```

Also, note the period of the end of each sentence :)